tests/components/plugwise/test_init.py

```diff
@@ -53,5 +55,38 @@ async def test_unload_entry(hass, mock_smile_adam):
     entry = await async_init_integration(hass, mock_smile_adam)
 
     mock_smile_adam.async_reset = AsyncMock(return_value=True)
-    assert await async_unload_entry(hass, entry)
+    assert await plugwise.async_unload_entry(hass, entry)
```

Comment:
this should use hass api for entry unloading

```suggestion
    await hass.config_entries.async_unload(entry.entry_id)
    await hass.async_block_till_done()
```

Reply:
Added import as well